src/data/characters.json

```diff
@@ -1,192 +1,220 @@
+    "rating": 5,
+    "id": "749031464883978310",
+    "percentX": 45
+  },
+  {
+    "name": "Klee",
+    "element": "Pyro",
+    "type": "character",
+    "rating": 5,
+    "id": "749031404964151376",
+    "percentX": 57
+  },
+  {
+    "name": "Bennett",
+    "element": "Pyro",
+    "type": "character",
+    "rating": 4,
+    "id": "749031351533043772",
+    "percentX": 58
+  },
+  {
+    "name": "Noelle",
+    "element": "Geo",
+    "type": "character",
+    "rating": 4,
+    "id": "749031431132545057",
+    "percentX": 58
+  },
+  {
+    "name": "Fischl",
+    "element": "Electro",
+    "type": "character",
+    "rating": 4,
+    "id": "749031374052393120",
+    "percentX": 59
+  },
+  {
+    "name": "Sucrose",
+    "element": "Anemo",
+    "type": "character",
+    "rating": 4,
+    "id": "749031457019658401",
+    "percentX": 60
+  },
+  {
+    "name": "Mona",
+    "element": "Hydro",
+    "type": "character",
+    "rating": 5,
+    "id": "749031417895452754",
+    "percentX": 58
+  },
+  {
+    "name": "Xiao",
+    "element": "Anemo",
+    "type": "character",
+    "rating": 5,
+    "id": "749031476665778347",
+    "percentX": 46
+  },
+  {
+    "name": "Beidou",
+    "element": "Electro",
+    "type": "character",
+    "rating": 4,
+    "id": "749031334273613904",
+    "percentX": 55
+  },
+  {
+    "name": "Ningguang",
+    "element": "Electro",
+    "type": "character",
+    "rating": 4,
+    "id": "749031423905759253",
+    "percentX": 58
+  },
+  {
+    "name": "Xiangling",
+    "element": "Pyro",
+    "type": "character",
+    "rating": 4,
+    "id": "749031470726905977",
+    "percentX": 60
+  },
+  {
+    "name": "Xingqiu",
+    "element": "Hydro",
+    "type": "character",
+    "rating": 4,
+    "id": "749031483263680563",
+    "percentX": 58
+  },
+  {
+    "name": "Chongyun",
+    "element": "Cryo",
+    "type": "character",
+    "rating": 4,
+    "id": "749031361934917682",
+    "percentX": 63
+  },
+  {
+    "name": "Qiqi",
+    "element": "Cryo",
+    "type": "character",
+    "rating": 5,
+    "id": "749031442494914661",
+    "percentX": 63
+  },
+  {
+    "name": "Keqing",
+    "element": "Electro",
+    "type": "character",
+    "rating": 5,
+    "id": "749031397896749127",
+    "percentX": 58
+  },
+  {
+    "name": "Tartaglia",
+    "element": "Hydro",
+    "type": "character",
+    "rating": 5,
+    "percentX": 60
+  },
+  {
+    "name": "Diona",
+    "element": "Cryo",
+    "type": "character",
+    "rating": 4,
+    "percentX": 57
+  },
+  {
+    "name": "Zhongli",
+    "element": "Geo",
+    "type": "character",
+    "rating": 5,
+    "percentX": 58
+  },
+  {
+    "name": "Xinyan",
+    "element": "Pyro",
+    "type": "character",
+    "rating": 4,
+    "percentX": 60
+  },
+  {
+    "name": "Albedo",
+    "element": "Geo",
+    "type": "character",
+    "rating": 5,
+    "percentX": 56
+  },
+  {
+    "name": "Ganyu",
+    "element": "Cryo",
+    "type": "character",
+    "rating": 5,
+    "percentX": 57
+  }
```

Comment:
So we can keep the percentX here for future banners when I create them automatically. However, we should have them on the items in the banner directly, I'll explain in the other files.